tests/org.codechecker.eclipse.rcp.unit.tests/resources/CodeChecker/bin/CodeChecker

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python
+import sys
+import argparse
+
+def checkers(args):
```

Comment:
Please add a docstring about emulating the checkers listing feature of codechecker.